test/cli/test_cli_functions.sh

```diff
@@ -120,12 +120,11 @@ function timeout {
 
 function install {
     test_noobaa install --mini
-    local status=$(kuberun get noobaa noobaa -o json | jq -r '.status.phase' 2> /dev/null)
```

Comment:
If we remove this line better replace it with `local status` to set this variable as local

Reply:
re-added